conda_env/cli/main_create.py

```
@@ -11,6 +11,10 @@
 from ..env import from_file
 from ..installers.base import get_installer, InvalidInstaller
 from .. import exceptions
+try:
+    from conda_env.utils import binstar as binstar_utils
+except:
```

We shouldn't catch all exceptions here.  This should be `except ImportError`.